experimental/packages/otlp-grpc-exporter-base/src/grpc-exporter-transport.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// NOTE: do not change these type imports to actual imports. Doing so WILL break `@opentelemetry/instrumentation-http`,
+// as they'd be imported before the http/https modules can be wrapped.
+import type { Metadata, ServiceError, ChannelCredentials } from '@grpc/grpc-js';
+import { ExportResponse } from './export-response';
+import { IExporterTransport } from './exporter-transport';
+
+const GRPC_COMPRESSION_NONE = 0;
+const GRPC_COMPRESSION_GZIP = 2;
```

Comment:
Where do these values come from?

Reply:
These are used internally by `@grpc/grpc-js` they're defined here so that we don't need to require `@grpc/grpc-js` in this file to get them.